lib/transaction/operation.go

```diff
@@ -16,6 +16,8 @@ const (
 	OperationPayment                            = "payment"
 	OperationCongressVoting                     = "congress-voting"
 	OperationCongressVotingResult               = "congress-voting-result"
+	OperationMembership                         = "membership"
```

Comment:
Doesn't look related

Reply:
@Geod24 Yes It does not related to this PR. 
I'll remove it.